tests/chainer_tests/functions_tests/array_tests/test_split_axis.py

```diff
@@ -1,4 +1,5 @@
 import unittest
+import warnings
```

Comment:
It seems that no changes are needed in `tests/chainer_tests/functions_tests/array_tests/test_split_axis.py`.

Reply:
Thanks, resolved.